code_search/README.md

```diff
@@ -68,47 +69,54 @@ $ python preprocess/scripts/process_github_archive.py -i files/select_github_arc
          --max-num-workers 16
 ```
 
-## 2. Function Summarizer
+## 2. Model Training
 
-This part generates a model to summarize functions into docstrings using the data generated in previous
-step. It uses `tensor2tensor`.
+A `Dockerfile` based on Tensorflow is provided along which has all the dependencies for this part of the pipeline. 
+By default, the based off Tensorflow CPU 1.8.0 for `Python3` but can be overridden in the Docker image build using
```

Comment:
nit: Grammer error "By default, the based off"

Reply:
Thanks for catching that.